__tests__/node.json

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
this is the AST output

Reply:
Is there a reason you’re checking this into the repository? I’d recommend doing tests with snapshots (see [`tests/arrays`](https://github.com/prettier/prettier/tree/master/tests/arrays) in the Prettier codebase for an example), and only checking that the output matches your expected output.

Reply:
yes definitely I'll add jest sooner but I wanted to have a ref of an AST just while I'm getting familiar w/ that structure as well as the prettier plugin architecture; this will be removed for sure